docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2794,3 +2794,119 @@ message Root
     uint32 column2 = 2;
 }
 ```
+
+## formatQuery
+
+Returns a formatted version of the given SQL query.
+
+**Syntax**
+
+```sql
+formatQuery(query)
+```
+
+**Arguments**
+
+- `query` - The SQL query to be formatted. [String](../../sql-reference/data-types/string.md)
+
+**Returned value**
+
+- The formatted query. [String](../../sql-reference/data-types/string.md).
+
+**Example**
+
+```sql
```

Comment:
(the example on l. 2843ff covers the function well enough, we don't need l. 2818 - 2841)